addon/components/upf-table/index.js

```diff
@@ -15,6 +15,7 @@ export default Component.extend({
   allRowsSelected: false,
   displayedColumnsPanel: false,
 
+  isPaginated: false,
```

Comment:
would have put it just after the `hasSortableColumns` so it stays with the table options.
+ had called it `hasPagination`, so that we keep the `hasX` naming :)